.github/CODEOWNERS

```diff
@@ -4,10 +4,27 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-# The @googleapis/yoshi-python is the default owner for changes in this repo
-*               @arithmetic1728 @sai-sunder-s @googleapis/googleapis-auth @googleapis/yoshi-python
+# The @googleapis/googleapis-auth and @googleapis/yoshi-python is the default owner for changes in this repo
+*                                                           @googleapis/googleapis-auth @googleapis/yoshi-python
+google/auth/_default.py                                     @googleapis/googleapis-auth @googleapis/aion-sdk
```

Comment:
There is no way to make both reviewers required for _default.py right?

Reply:
Doesn't seem like that is supported https://github.com/isaacs/github/issues/1205